StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp8/NamingRules/SA1316CSharp8UnitTests.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+namespace StyleCop.Analyzers.Test.CSharp8.NamingRules
+{
+    using StyleCop.Analyzers.Test.CSharp7.NamingRules;
+
+    public class SA1316CSharp8UnitTests : SA1316CSharp7UnitTests
```

Comment:
Was this just missed by accident?

Reply:
Yes, as SA1316 doesn't have any tests for CSharp 6, I probably missed it when adding the C# 8 tests.